finite_state_sdk/__init__.py

```diff
@@ -1968,6 +1974,7 @@ def search_sbom(token, organization_context, name=None, version=None, asset_vers
     return records
 
 
+@retry(stop=stop_after_attempt(10), wait=wait_fixed(10), retry=retry_if_exception(is_not_breakout_exception))
```

Comment:
i think this is too many retries, maybe 3? 5? how long does the test take to run?

Reply:
Changed to 5